chapter02/jxp/ch2ex1.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use File::Spec;
+use Cwd qw ( abs_path );
+
+my $curdir=File::Spec->curdir();
+foreach my $file ( my @files=<$curdir/*> ){
```

Comment:
The temporary assignment to the `@files` array isn't necessary. Prepending the glob with the current directory is also unnecessary, `<*>` will expand to filenames in the current working directory.